modules/uhttp/filters_test.go

```diff
@@ -42,14 +42,58 @@ import (
 	"github.com/uber/jaeger-client-go/config"
 )
 
-func TestFilterChain(t *testing.T) {
+func TestDefaultFiltersWithNopHost(t *testing.T) {
+	// setup
 	host := service.NopHost()
+	stats.SetupHTTPMetrics(host.Metrics())
+
+	t.Run("run parallel", func(t *testing.T) {
```

Comment:
How is it parallel? It looks like 3 sequential runs, I think t.Run is sync..
May be I am missing something, but it looks like you've forgot to set t.Parallel()?